xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/WinVideoFilter.h

```diff
@@ -96,7 +96,7 @@ class COutputShader : public CWinShader
   unsigned m_sourceHeight = 0;
   int m_lutSize = 0;
   int m_ditherDepth = 0;
-  int m_toneMappingMethod = VS_TONEMAPMETHOD_REINHARD;
+  int m_toneMappingMethod = VS_TONEMAPMETHOD_HABLE;
```

Comment:
Implicit inclusion of cores/VideoSettings.h again

Reply:
Replaced with:

`int m_toneMappingMethod = 0;`

Here is not need define default value because always tonemap is activated is passed current method and param.

`COutputShader::SetToneMapParam(int method, float param);`